.travis.yml

```diff
@@ -32,3 +32,4 @@ script:
   # push docker images to registry if master branch:
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"
   - make push -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master" -e IMAGE_PREFIX="docker.io/avenga"
+  - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then make build-all push-all -e CURRENT_BRANCH="$BRANCH" -e ONLY_BRANCH="master"; fi
```

Comment:
Build and push should go the avenga and 7val Docker-namespace.Also a test-all target is missing